src/assets/scss/main.css

```diff
@@ -117,4 +117,8 @@
 .Terms p span {
   font-weight: 600;
   color: #242424;
+}
+
+.Family-Details {
```

Comment:
BEM does not follow camel case. instead of .Family-Details the class name should be .family-details. Make sure you fix this in other places as well.